cmd/swarmd/main.go

```diff
@@ -238,6 +251,7 @@ func init() {
 	mainCmd.Flags().String("listen-debug", "", "Bind the Go debug server on the provided address")
 	mainCmd.Flags().String("listen-metrics", "", "Listen address for metrics")
 	mainCmd.Flags().String("join-addr", "", "Join cluster with a node at this address")
+	mainCmd.Flags().String("generic-node-resource", "", "user defined resources (e.g. fpga=2;gpu={UUID1,UUID2,UUID3})")
```

Comment:
Corrected to add an 's'
However, @aluzzardi commented the following on the service command:

> nit: this is a service command, have service in the flag isn't necessary (can be shortened to --generic-resource)

I'm not sure what to do for that :)
I understand your argument @dongluochen which is that it might be confusing to the user but on the other side there is a help and an error message.

What do you think ?